Src/LiquidProjections.RavenDB/RavenChildProjector.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LiquidProjections.RavenDB
+{
+    public class RavenChildProjector<TProjection> : IRavenChildProjector
+        where TProjection : class, IHaveIdentity, new()
+    {
+        private readonly RavenEventMapConfigurator<TProjection> mapConfigurator;
+
+        public RavenChildProjector(
+            IEventMapBuilder<TProjection, string, RavenProjectionContext> mapBuilder,
+            IEnumerable<IRavenChildProjector> children = null,
+            IProjectionCache cache = null,
+            string collectionName = null)
+        {
+            mapConfigurator = new RavenEventMapConfigurator<TProjection>(mapBuilder, children, cache, collectionName);
+        }
+
+        Task IRavenChildProjector.ProjectEvent(object anEvent, RavenProjectionContext context)

```

Comment:
Can't we use have this method as a *public* one on the generic projector as well? This would avoid the entire concept of a separate parent and child projector